Training/Training/SortAndSearch.c

```diff
@@ -0,0 +1,28 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Gokul TS @Metamation India.
+// ------------------------------------------------------------------
+// SortAndSearch.c
+// C Program for Bubble sort,Insertion sort and binary search algorithm
+// ------------------------------------------------------------------------------------------------
+#include "SortAndSearch.h"
+
+void InsertionSort (int arr[], int size) {
+   for (int i = 1; i < size; i++) {
+      int key = arr[i], j = i - 1;
+      while (j >= 0 && arr[j] > key)
+         arr[j + 1] = arr[j--];
+      arr[j + 1] = key;
+   }
+}
+
+int BinarySearch (int sortArr[], int key, int size) {
+   int low = 0, high = size - 1, result = -1;
+   while (low <= high) {
+      int mid = low + (high - low) / 2;
+      if (sortArr[mid] == key) result = mid;

```

Comment:
Your program will continue to search through the array even after it has found the element. This is not Binary Search.